stats/system_tag_set.go

```diff
@@ -0,0 +1,87 @@
+package stats
+
+import (
+	"fmt"
+)
+
+// This code is borrowed from "enumer -type=SystemTagSet -transform=snake -trimprefix=Tag"
+// with modification to TagSubProto to be represent as subproto instead of sub_proto.
+//
+// The original code gen is "protosub_proto...", here we want sub_proto to become subproto,
```

Comment:
Also, when you go back to go:generating this file, can you do it with a filename that ends on `_gen.go`. To avoid linter issues, I think we should give them common filenames and add these as an exclusion in a lot of the linter rules, i.e. implement this `TODO`: https://github.com/loadimpact/k6/blob/ce85f802fe139b36e41560c5acdd0478ab3eb375/.golangci.yml#L20

Reply:
@na-- I'm a little confused. So what's the final decision? Your original comment makes me think that you don't want to break backward compatible, i.e we can't use `TagSubProto` with `enumer`, we can't do that right now with any existing tool.

Reply:
Instead of `TagSubProto`, call the Go constant `TagSubproto` and enumer would work perfectly fine with it as it is

Reply:
@na-- Ah right, fixed.